lib/astarte_rpc/config.ex

```diff
@@ -68,6 +68,17 @@ defmodule Astarte.RPC.Config do
     type: :integer,
     default: 0
 
+  @envdoc ""
+  app_env :amqp_connection_ssl_enabled, :astarte_rpc, :amqp_connection_ssl_enabled,
+    os_env: "RPC_AMQP_CONNECTION_SSL_ENABLED",
+    type: :boolean,
+    default: false
+
+  @envdoc "Specifies the certificates of the root Certificate Authorities to be trusted."
```

Comment:
Add "Defaults to a path to the cURL certificate bundle."

Reply:
I think it's pretty misleading, especially given the user is running this in a container in 99% of cases. I'd go for something like this:

`When not specified, the bundled cURL certificate bundle will be used.`